C/other/concatenate_two_strings.c

```diff
@@ -0,0 +1,30 @@
+// The best way to concatenate two strings in C programming is by using the strcat() function.But,in this example, we will concatenate two strings manually.
+#include <stdio.h>
+#include <string.h>     //header file
+
+int main() {
+  char str1[100] = "Hey,Open Source is amazing", str2[] = " Start contributing";   //It's important to note that the length of str1 should be sufficient to hold the string after concatenation.
```

Comment:
- Remove the redundant comments.
- Don't fix input from yourself. Read I/P from user for both the strings.